source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -75,15 +75,14 @@ AndroidContext: class extends OpenGLContext {
 		fence := this createFence()
 		fence sync()
 		eglImage := gpuRgba as EGLBgra
-		sourcePointer := eglImage buffer lock(false)
+		sourcePointer := eglImage buffer lock(false) as UInt8*
```

Comment:
What happened, why you need to cast now ? Or should it be there from the start ?